plugins/exporter.koplugin/main.lua

```diff
@@ -102,6 +102,7 @@ local Exporter = WidgetContainer:extend{
         flomo = require("target/flomo"),
         html = require("target/html"),
         joplin = require("target/joplin"),
+        nextcloud_notes = require("target/nextcloud_notes"),
```

Comment:
I think it's alphabetical. ;-)

Reply:
Makes sense, [fixed](https://github.com/koreader/koreader/pull/12301/commits/c873dd5624e9c4470ceee8699b2911004ccad46a). Thanks!